### PR TITLE
docs(tutorials): update Node requirement

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Remix doesn't support Node v12

### CODE REVIEW

**Comment by machour** [comment]
Related to #3658

File: docs/tutorials/blog.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ Click this button to create a [Gitpod][gitpod] workspace with the project set up
 
 If you want to follow this tutorial locally on your own computer, it is important for you to have these things installed:
 
-- [Node.js][node-js] version (^12.22.0, ^14.17.0, or >=16.0.0)
+- [Node.js][node-js] version (^14.17.0, or >=16.0.0)
 - [npm][npm] 7 or greater
-- A code editor
+- A code editor ([VSCode][vs-code] is a nice one)
```
Comment (machour) [inline docs/tutorials/blog.md:24]: WebStorm also, but I'm not sure this belongs in this PR 🙈

**Review by machour — APPROVED** [review]
We do indeed specify it in our package.json files:

```json
  "engines": {
    "node": ">=14"
  }
```

**Review by kentcdodds — APPROVED** [review]
Thanks 👍

File: docs/tutorials/blog.md (hunk 1 above)
Comment (MichaelDeBoey) [reply docs/tutorials/blog.md:24]: I just copied this over from the jokes tutorial since I saw a slight difference here

**Pull request merged**